.github/workflows/upload-content-files.yml

```diff
@@ -8,6 +8,8 @@ on:
   repository_dispatch:
     types:
       - forms_changed
+      - next_steps_changed
+      # - answer_filter_changed #( Not sure if we can separate updating a question to include an answer filter from saving changes on the entire form )
```

Comment:
We can't separate answer filters so will remove this type.

Reply:
Does this mean that forms_changed triggers when the filters have changed?

Reply:
From my understanding yes. 
We have to publish the form for the answer filters on a question to be published (even though the form itself might not be ready to release).

Reply:
yes, it does, because answer filters are stored _inside_ form fields themselves